shared/src/main/scala/tastyquery/ast/Trees.scala

```diff
@@ -17,7 +19,7 @@ object Trees {
     def unapply(e: TypeComputationError): Option[Tree] = Some(e.tree)
   }
 
-  abstract class Tree {
+  abstract class Tree(val span: Span) {
```

Comment:
To be honest, I think `Span` is better because it implies that it consists of a start and an end, while `Position` sounds like it's a single point of position, like the `Coord` in dotc. If I understand correctly, we won't keep `Coord` in the code?

Reply:
Hum. Maybe you're right. Let's keep `span` for now.

Reply:
on this point, if we choose to have some span attached to definitions then `Coord` is how you link to things without trees, such as scala 2 symbols / java symbols